rust/gui-client/src-tauri/src/client/deep_link/linux.rs

```diff
@@ -0,0 +1,34 @@
+//! TODO: Not implemented for Linux yet
+
+use super::Error;
+use connlib_shared::control::SecureUrl;
+use secrecy::Secret;
+
+pub(crate) struct Server {}
+
+impl Server {
+    pub(crate) fn new() -> Result<Self, Error> {
+        tracing::warn!("Not implemented yet");
+        tracing::trace!(scheme = super::FZ_SCHEME, "prevents dead code warning");
+        // Stop Cargo from erroring
```

Comment:
what's the error here?

Reply:
Dead code. It should go away once I allow dead code for that Error enum